.github/workflows/deploy.yml

```diff
@@ -28,5 +28,4 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
-    - run: npm run zapier-test
-    
\ No newline at end of file
+    - run: npm run test
```

Comment:
The `test` command calls `build` on its own, because you cannot run the tests without building it first. Adding a publishing pipeline is a good idea. I will create an issue for that.

Reply:
The `test` command no longer needs the `build` command so I added it to the workflow.